src/main/java/io/jenkins/plugins/pipelinegraphview/utils/AbstractPipelineNode.java

```diff
@@ -33,8 +33,8 @@ public long getStartTimeMillis() {
         return timingInfo.getStartTimeMillis();
     }
 
-    public long getTotalDurationMillis() {
-        return totalDurationMillis;
+    public Long getTotalDurationMillis() {
+        return state == PipelineState.RUNNING ? null : totalDurationMillis;
```

Comment:
are there any other states I should account for 🤔

Reply:
paused? although i don't think we actually set that currently